Software/Python/easygopigo3.py

```diff
@@ -896,6 +910,9 @@ def __init__(self, port, pinmode, gpg):
 
         """
         debug("Sensor init")
+
+        if type(gpg) != EasyGoPiGo3:
```

Comment:
good change here, although I think it could be a GoPiGo3 object too.  EasyGoPiGo3 inherits from GoPiGo3.